### ISSUE TITLE
Improve resizing experience

### ISSUE THREAD

**Issue by Photon89** [shutter-project/shutter#14]
Improve the resize dialog it offers a live preview / positioning (ala gimp), and allow it to be called by a keybinding (make this seem a core feature too).

Launchpad Details: [#LP344999](https://bugs.launchpad.net/bugs/344999) Vadim Peretokin - 2009-03-18 19:24:04 +0000

**Comment by Flimm** [comment]
Where is this resize dialog? I'm trying to resize a screenshot, and I don't find this feature. I have a HiDPI set up, and I would like the saved screenshot to be resized to smaller size, for inclusion in an email.

**Comment by Photon89** [comment]
It's a plugin, accessible via _Screenshot > Run a plugin... > Resize_

**Comment by SjoerdV** [comment]
Eventually found it ;-). OK, just assume that **cropping** and **resizing** are THE most common action one could perform on an image. Why is 'Cropping' only available inside the Built-in Editor and the 'Resizing' in the Main 'Gallery' screen? I would like to see Resizing as an option inside the Built-in Editor as well. Just my 2 cents.

**Comment by Photon89** [comment]
It also works in the editor, actually. The only problem is just that you need to first resize the canvas and then resize the actual image. That's indeed not very comfortable.

**Comment by SjoerdV** [comment]
> 

Ah like that! Maybe something that can be improved upon then. I must say I love Shutter so far. It seems to be one of the few (maybe the only one) screenshot tools that doesn't suffer from screen scaling issues on 4k screens.

**Comment by Photon89** [comment]
Absolutely, the editor has quite some quirks which need to be worked out! I fear, currently, we only have manpower for keeping Shutter afloat, that is, no functional changes... Any contributions are welcome though!
